history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -272,11 +296,13 @@ we do humbly hope that the king`'s tenderness will appear and extend,
 with his power to express the same; recommending to his princely clemency,
 the case of our present suffering friends hereunto annexed.
 
+[.letter-heading]
 TO THE KING.
 
 The distressed Case and request of the suffering people commonly called Quakers,
 humbly presented.
 
+[.salutation]
```

Comment:
And also, let's take slight liberty here and just make this the first word of the next paragraph, so that it reads:

```ASCIIDOC
Showing, that according to <etc...>
```